### PR TITLE
Change ECIP-1000 from mediawiki to markdown

### PR DESCRIPTION
**Pull request by TokenHash**
• fixed titles
• fixed bold and shaded terms
• fixed process.png image
• migrated from mediawiki format to markdown

### CODE REVIEW

File: ECIPs/ECIP-1000.md [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+
+An Ethereum Classic Improvement Proposal (ECIP) is a design document providing information to the Ethereum Classic community, or describing a new feature for Ethereum Classic or its processes or environment. The ECIP should provide a concise technical specification of the feature and a rationale for the feature.
+
+We intend ECIPs to be the primary mechanism for proposing new features, for collecting community input on an issue, and for documenting the design decisions that have gone into Ethereum Classic. The **ECIP author** is responsible for building consensus within the community and documenting dissenting opinions.
+
+Because the ECIPs are maintained as text files in a versioned repository, their revision history is the historical record of the feature proposal.
+
+# Copyright
+
+This ECIP is licensed Apache-2, originally by [Luke Dashjr](https://github.com/luke-jr) under BSD 2-clause license.
```
Comment (realcodywburns) [inline ECIPs/ECIP-1000.md]: Is this right? Does luke-jr have a copyright over all improvement processes?

Comment (realcodywburns) [reply ECIPs/ECIP-1000.md]: if so, we need to link to the real license

Comment (YazzyYaz) [reply ECIPs/ECIP-1000.md]: To confirm, is this line part of the ECIP by @TokenHash?

As far as I can tell, he just switched file formats to Markdown and then some basic boldening and title fixes. We can create a new issue for this specific point.

**Comment by YazzyYaz** [comment]
In terms of the licensing issue, it should be a separate ticket. What @TokenHash did here was just change the file format to Markdown and some basic formatting, not modify the actual contents of the ECIP doc.

File: ECIPs/ECIP-1000.md (hunk 1 above)
Comment (TokenHash) [reply ECIPs/ECIP-1000.md]: yes, this is the original ECIP-1000 which I adjusted for formatting and alinement with:

• https://github.com/ethereumclassic/ECIPs/blob/master/README.md
• https://github.com/ethereumclassic/ECIPs/blob/master/ECIP-1.sample.md
• https://github.com/ethereumclassic/ECIPs/blob/master/ECIP-number.template.md
• https://github.com/ethereumclassic/ECIPs/blob/master/process.png

Also adjusted the status key:

• Draft
• Last call
• Accepted
• Final
• Deferred
• Rejected
• Withdrawn
• Replaced

But, I did not change content, basic structure or licensing.

**Pull request merged**